build-tools/packages/version-tools/src/test/internalVersionScheme.test.ts

```diff
@@ -387,4 +388,60 @@ describe("internalScheme", () => {
 			);
 		});
 	});
+
+	describe("checking test version scheme", () => {
+		it("0.0.0-285010-test is a valid test version", () => {
+			const input = `0.0.0-285010-test`;
+			const result = isInternalTestVersion(input);
+			assert.isTrue(result);
+		});
+
+		it("0.0.0-test-285010 is not a valid test version", () => {
```

Comment:
Do we produce version strings like this?

Reply:
we do not. The test versions are always `0.0.0-123456-test`